apps/browser-tests/tsconfig.json

```diff
@@ -0,0 +1,27 @@
+{
+  "$schema": "https://json.schemastore.org/tsconfig",
```

Comment:
Would this be common with the browser-wallets tsconfig? Maybe we can extracthis on the root, import them, then just add those that are specific.